application/templates/cms/create_chart_2.html

```diff
@@ -259,40 +253,75 @@ <h2 class="heading-medium">4. Component chart options</h2>
                     </form>
                 </div>
 
+                {#
+                    LINE CHARTS _______________________________
+                    Must have ETHNICITY as their series - we only need to set a column for the x-axis
+                #}
+                <div id="line_graph_options" class="chart-builder-section chart-option-group grid-row"
+                     style="display:none">
+                    <h2 class="heading-medium builder-section-number">Line chart options</h2>
+                    <form>
+                        <div>
+                            <label for="line__x-axis_column">Select a value for the horizontal (X) Axis.<br>Choose 'Time' for time-series data.</label>
+                            <select id="line__x-axis_column" class="form-control">
+                                <option>Time</option>
+                            </select>
+                        </div>
+                    </form>
+                </div>
 
                 {#
-                    PANEL BAR CHARTS _______________________________
+                    PANEL LINE CHARTS _______________________________
+                    Are not more complex than regular line charts
```

Comment:
I don't think this line of comment is very useful? But if we want it then `no more complex` rather than ` not more complex`.

Reply:
I'm not sure it's that useful either, but all I've done here is re-order blocks of code. Haven't updated any comments. And I'm not sure I want to make the decision to rip all of the code comments out here. Updating `not` to `no` is fine, I guess, but seems not important enough to justify a commit of its own?